cmd/gluster-virtblock/main.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	gvb "github.com/gluster/gluster-csi-driver/pkg/gluster-virtblock"
+	"github.com/gluster/gluster-csi-driver/pkg/gluster-virtblock/config"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	// #nosec
```

Comment:
what is `#nosec`

Reply:
It's an annotation to skip golint warning as error check is not done for now.